proto/proxy/destination/destination.proto

```diff
@@ -75,11 +75,26 @@ message WeightedAddrSet {
 }
 
 message WeightedAddr {
-  common.TcpAddress addr = 1;
-  uint32 weight = 3;
-  map<string, string> metric_labels = 4;
-}
+    common.TcpAddress addr = 1;
+    uint32 weight = 3;
+    map<string, string> metric_labels = 4;
+    TlsVerification tls_verification = 5;
+  }
 
+  // Which strategy should be used for verifying TLS.
+  message TlsVerification {
+      oneof strategy {
+          // Verify the certificate based on the Kubernetes pod name, and ensure
+          // that the pod is configured with the same Conduit control plane
+          // namespace as this proxy.
+          K8sPodNamespace k8s_pod_namespace = 1;
+      }
+  }
+
+  message K8sPodNamespace {
```

Comment:
wdyt about nesting this type within `TlsVerification` (since it doesn't make sense outside of that context)? Then, I'd move the docstring from the field onto the message type.

Reply:
Yeah, that sounds correct, will do.